internal/bundler/linker.go

```diff
@@ -5850,6 +5850,16 @@ func (c *linkerContext) generateSourceMapForChunk(
 	dataForSourceMaps []dataForSourceMap,
 	canHaveShifts bool,
 ) (pieces sourcemap.SourceMapPieces) {
+    var sourceAbsDir string
+    if c.options.SourceBase != "" {
+        var succes bool
+        sourceAbsDir, succes = c.fs.Abs(c.options.SourceBase)
+        if !succes {
+            panic("Could not determine source base")
+        }
+    } else {
+        sourceAbsDir = chunkAbsDir
+    }
```

Comment:
If there is a better way to do this if statement let me know. This is the first time writing go